integration-tests/testconfig/keeper/keeper.toml

```diff
@@ -2,6 +2,58 @@
 [Common]
 chainlink_node_funding = 0.5
 
+[NodeConfig]
+BaseConfigTOML = """
+[Feature]
+FeedsManager = true
+LogPoller = true
+UICSAKeys = true
+
+[Log]
+Level = 'debug'
+JSONConsole = true
+
+[Log.File]
+MaxSize = '0b'
+
+[WebServer]
+AllowOrigins = '*'
+HTTPPort = 6688
+SecureCookies = false
+HTTPWriteTimeout = '1m'
+SessionTimeout = '999h0m0s'
+
+[WebServer.RateLimit]
+Authenticated = 2000
+Unauthenticated = 1000
+
+[WebServer.TLS]
+HTTPSPort = 0
+
+[Database]
+MaxIdleConns = 20
+MaxOpenConns = 40
+MigrateOnStartup = true
+
+[OCR2]
+Enabled = true
+
+[Keeper]
+TurnLookBack = 0
+
+[Keeper.Registry]
+PerformGasOverhead = 150000
+SyncInterval = '5m0s'
+
+[P2P]
```

Comment:
This part did not exist in the test config previously. P2P is not needed. At least not for keepers v1 smoke test.